src/dapp/components/farm/Gold.tsx

```diff
@@ -131,10 +132,12 @@ export const Gold: React.FC<Props> = ({ inventory }) => {
             className={classnames("gather-tree", {
               "gatherer-selected": isHighlighted,
               gatherer: isNextToChop,
+              "game-object": true,
```

Comment:
Rather than putting these in the object with `true` value, you can pass them as a string along with `gather-tree`. 

```
classnames('gather-tree', 'game-object', 'gold', { ... })
```

Reply:
@spencerdezartsmith this is resolved in the latest commit! 😁